torch/csrc/jit/codegen/cuda/arith.cpp

```diff
@@ -293,6 +293,16 @@ TensorView* unaryOp(UnaryOpType type, TensorView* v1) {
   return unaryOp(type, v1->as<Val>())->as<TensorView>();
 }
 
+Val* unaryOp(UnaryOpType type, Val* v, DataType dtype) {
```

Comment:
Should this be renamed to `unaryIsOp`, since you're hard coding the out type and it would be more consistent?

Reply:
Yes, fixed